data/chameleoncloud/sites/uc/clusters/chameleon/nodes/10ad276b-ba2f-4fe7-bb0d-203b6ff84432.json

```diff
@@ -5,123 +5,122 @@
+      "model": "Ethernet Controller E810-C for SFP",
       "rate": 25000000000,
-      "vendor": "Intel Corporation"
+      "vendor": "Intel"
     },
     {
-      "device": "NIC.Integrated.1-2",
+      "device": "eno12409",
+      "driver": "ice",
       "enabled": true,
       "interface": "Ethernet",
       "mac": "40:a6:b7:83:83:29",
-      "management": false,
-      "model": "Intel(R) 25GbE 4P E810-XXV OCP",
+      "model": "Ethernet Controller E810-C for SFP",
       "rate": 25000000000,
-      "vendor": "Intel Corporation"
+      "vendor": "Intel"
     },
     {
-      "device": "NIC.Integrated.1-3",
+      "device": "eno12419",
+      "driver": "ice",
       "enabled": false,
       "interface": "Ethernet",
       "mac": "40:a6:b7:83:83:2a",
-      "management": false,
-      "model": "Intel(R) 25GbE 4P E810-XXV OCP",
-      "vendor": "Intel Corporation"
+      "model": "Ethernet Controller E810-C for SFP",
+      "rate": 25000000000,
+      "vendor": "Intel"
     },
     {
-      "device": "NIC.Integrated.1-4",
+      "device": "eno12429",
+      "driver": "ice",
       "enabled": false,
       "interface": "Ethernet",
       "mac": "40:a6:b7:83:83:2b",
-      "management": false,
-      "model": "Intel(R) 25GbE 4P E810-XXV OCP",
-      "vendor": "Intel Corporation"
+      "model": "Ethernet Controller E810-C for SFP",
+      "rate": 25000000000,
+      "vendor": "Intel"
     },
     {
-      "device": "InfiniBand.Slot.3-1",
+      "device": "eno8303",
+      "driver": "tg3",
+      "enabled": false,
+      "interface": "Ethernet",
+      "mac": "d0:8e:79:f7:9f:ba",
+      "model": "NetXtreme BCM5720 Gigabit Ethernet PCIe",
+      "rate": 1000000000,
+      "vendor": "Broadcom"
+    },
+    {
+      "device": "eno8403",
+      "driver": "tg3",
+      "enabled": false,
+      "interface": "Ethernet",
+      "mac": "d0:8e:79:f7:9f:bb",
+      "model": "NetXtreme BCM5720 Gigabit Ethernet PCIe",
+      "rate": 1000000000,
+      "vendor": "Broadcom"
+    },
+    {
+      "device": "ibp75s0",
+      "driver": "mlx5_core[ib_ipoib]",
       "enabled": false,
       "interface": "InfiniBand",
-      "mac": "e8:eb:d3:08:53:b6",
-      "management": false,
-      "model": null,
-      "vendor": "Mellanox Technologies"
+      "mac": "e8:eb:d3:03:00:08:53:b6",
+      "model": "MT28908 Family [ConnectX-6]",
+      "vendor": "Mellanox"
     }
   ],
   "node_name": "J0BG3Q3",
   "node_type": "gpu_a100_pcie",
-  "placement": {
-    "node": "1",
-    "rack": ""
-  },
+  "placement": {},
   "processor": {
-    "cache_l1": 3200000,
-    "cache_l2": 51200000,
-    "cache_l3": 61440000,
+    "cache_l1d": 49807,
+    "cache_l1i": 32768,
+    "cache_l2": 1310720,
+    "cache_l3": 62914560,
     "clock_speed": 2300000000,
-    "instruction_set": "x86-64",
+    "instruction_set": "x86_64",
     "model": "Intel(R) Xeon(R) Platinum 8380 CPU @ 2.30GHz",
-    "vendor": "Intel",
-    "version": "Model 106 Stepping 6"
```

Comment:
it's unfortunate, but OK that we lose this info.

Although we're parsing out the results of various cpu info tools now, it's probably ok to instead use the cpuid info to look up the processor from a DB, since all cpus with same CPUID should have same cache, clock speed, model, stepping, etc....